### ISSUE TITLE
Don't work with MediaWiki 1.34

### ISSUE THREAD

**Issue by Xylon2** [jthingelstad/foreground#353]
### Setup

- MediaWiki version: 1.34
- PHP version: 7.2.26
- Foreground version: 2.2.0

### Issue

So I'm getting this error:
2020/01/01 15:56:09 [error] 787#100085: *18 FastCGI sent in stderr: "PHP message: PHP Fatal er
ror:  Uncaught Error: Call to undefined function wfSuppressWarnings() in /usr/local/www/savvy_
mediawiki/skins/foreground/Foreground.skin.php:63

When I look in release notes for MediaWiki 1.34 I see:
The global functions wfSuppressWarnings and wfRestoreWarnings, deprecated in 1.26, have been removed.

**Comment by kghbln** [comment]
### Issue
Also reported [on mediawiki.org](https://www.mediawiki.org/wiki/Topic:Veuhjsm3g02gzhuk)
```
Fatal error: Uncaught Error: Call to undefined function wfSuppressWarnings() in /../w/skins/foreground/Foreground.skin.php:60
```
### Stack trace:
```
#0 /../w/includes/skins/SkinTemplate.php(217): foregroundTemplate->execute()
#1 /../w/includes/OutputPage.php(2574): SkinTemplate->outputPage()
#2 /../w/includes/exception/MWExceptionRenderer.php(134): OutputPage->output()
#3 /../w/includes/exception/MWExceptionRenderer.php(53): MWExceptionRenderer::reportHTML(Object(Error))
#4 /../w/includes/exception/MWExceptionHandler.php(121): MWExceptionRenderer::output(Object(Error), 2)
#5 /../w/includes/exception/MWExceptionHandler.php(195): MWExceptionHandler::report(Object(Error))
#6 /../w/includes/exception/MWExceptionHandler.php(167): MWExceptionHandler::handleException(Object(Error))
#7 [internal function]: MWExceptionHandler::handleUncaughtException(Object(Error))
#8 in /../w/skins/foreground/Foreground.skin.php on line 60
```

**Comment by kghbln** [comment]
@Hutchy68 It will be cool if you could have a look at this. For not I will document that the skin is not compatible with MW 1.34+

**Comment by Hutchy68** [comment]
I'll merge some commits and get another release out tomorrow.

**Comment by kghbln** [comment]
> I'll merge some commits and get another release out tomorrow.

You are a hero! :)

**Comment by Hutchy68** [comment]
Released v2.2.3

I'll update some of the other PRs as some are redundant and/or not needed.

**Comment by kghbln** [comment]
Utterly cool. Thanks a lot. Will update the docu now.
